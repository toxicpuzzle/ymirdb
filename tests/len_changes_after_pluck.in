SET b 1 2 
SET a 1 2 3 4 5 6 b
LEN a
PLUCK a 7
LEN a
bye