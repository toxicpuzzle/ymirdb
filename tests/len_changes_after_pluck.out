> ok

> ok

> 8

> b

> 6

> bye
